### ISSUE TITLE
Transformer reproduce issue with InterHuman

### ISSUE THREAD

**Issue by Noperi0r** [gohar-malik/InterMask#10]
Hello, I'm trying to reproduce intermask model and I'm having some issues with it currently.
I tested reproducing on one H200 gpu environments which provides over 80 GB vram compared A100.

Result on h200 environment with interhuman dataset was the following:
```
========== MM Distance Summary ==========
---> [ground truth] Mean: 3.7856 CInterval: 0.0007
---> [best_fid.tar] Mean: 3.7889 CInterval: 0.0012
========== R_precision Summary ==========
---> [ground truth](top 1) Mean: 0.4168 CInt: 0.0041;(top 2) Mean: 0.5993 CInt: 0.0043;(top 3) Mean: 0.7014 CInt: 0.0056;
---> [best_fid.tar](top 1) Mean: 0.4537 CInt: 0.0056;(top 2) Mean: 0.6131 CInt: 0.0062;(top 3) Mean: 0.6945 CInt: 0.0060;
========== FID Summary ==========
---> [ground truth] Mean: 0.2901 CInterval: 0.0062
---> [best_fid.tar] Mean: 5.3931 CInterval: 0.0749
========== Diversity Summary ==========
---> [ground truth] Mean: 7.7718 CInterval: 0.0221
---> [best_fid.tar] Mean: 7.9761 CInterval: 0.0282
========== MultiModality Summary ==========
---> [best_fid.tar] Mean: 2.3364 CInterval: 0.0859
```
The evaluation results show that MM Distance, R precision value is higher than ones in the paper, but the FID value is worse than that. I wanted to reproduce it as the paper shows.

On the other hand, the result below shows the evaluation values using Inter-X.
```
========== MM Distance Summary ==========
---> [ground truth] Mean: 3.5742 CInterval: 0.0107
---> [best_fid.tar] Mean: 3.6678 CInterval: 0.0146
========== R_precision Summary ==========
---> [ground truth](top 1) Mean: 0.4353 CInt: 0.0051;(top 2) Mean: 0.6281 CInt: 0.0037;(top 3) Mean: 0.7365 CInt: 0.0037;
---> [best_fid.tar](top 1) Mean: 0.4083 CInt: 0.0048;(top 2) Mean: 0.5985 CInt: 0.0037;(top 3) Mean: 0.7073 CInt: 0.0036;
========== FID Summary ==========
---> [ground truth] Mean: 0.0012 CInterval: 0.0001
---> [best_fid.tar] Mean: 0.4066 CInterval: 0.0123
========== Diversity Summary ==========
---> [ground truth] Mean: 8.9474 CInterval: 0.0500
---> [best_fid.tar] Mean: 8.9539 CInterval: 0.0570
========== MultiModality Summary ==========
---> [best_fid.tar] Mean: 1.9654 CInterval: 0.0637
```
I can say it's been reproduced when using Inter-X because MM Dist, R Precision, FID values all show the simliar or a bit better values than paper, and this makes me more confused because its result is different to InterHuman's.

Can you guess the reason why the training cannot reproduce on InterHuman dataset? 
I read your response of previous issues and as I understand you were using one A100 to train transformer with 52 batch and 500 epoch. 
Also I run the training on H200 environment, with default value but seed value by changing it to 0, since the evaluation result was way far better than defualt seed value 3407. Besides your pretrained trans_default used seed value 0.

Did you preprocess something with InterHuman dataset or can there be possibilities where issues arise because of the InterHuman dataset itself, rather than because of the Intermask code? Because I found out there are some overlapped files in the annots directory so for example, 3407.txt and 3407(1).txt both exist. I deleted the overlapped files like 3407(1).txt, remaining unique file per sequence but I don't think it critically matters for reproducing.  

It'd be so grateful if you give me some advice or solutions on this issue please!

**Comment by yeminglang** [comment]
Hey, how long does it take for an H200 GPU to train the Inter-human and Inter-X datasets?

**Comment by Gdw040199** [comment]
@Noperi0r  Could you please tell me which version of the transformer weights you are using for your test (I'm referring to best_acc.tar or latest.tar)? Because according to the open-source script, these are the only ones that can be saved during the training process. The weights they have made available (best_fid.tar) might be the result after being renamed.

**Comment by Noperi0r** [comment]
@yeminglang Sorry for late comment, Interhuman took nearly over 40 hours, and 20 hours around for Inter-X's.

**Comment by Noperi0r** [comment]
@Gdw040199 I was using best_fid and it can be saved by addiing --do_eval argument when starting trainng code.
